src/plugins/linea.eth/ponder.config.ts

```
@@ -24,6 +24,7 @@ export const config = createConfig({
     linea: {
       chainId: linea.id,
       transport: http(process.env[`RPC_URL_${linea.id}`]),
```

Suggest adding a helper function for this that all our plugins could use.

The helper function would get the RPC URL for a given chainId and work similar to `rpcRequestRateLimit`.

Importantly: If no environment variable is set for a RPC URL that we expect this utility function should throw a helpful error message that the expected environment variable is missing.